app/models/organization.rb

```diff
@@ -12,15 +12,14 @@ class Organization < ApplicationRecord
   acts_as_followable
 
   has_many :api_secrets, through: :users
-  has_many :articles
-  has_many :listings
-  has_many :collections
-  has_many :credits
-  has_many :display_ads
-  has_many :notifications
+  has_many :articles, dependent: :nullify
+  has_many :listings, dependent: :nullify
+  has_many :collections, dependent: :nullify
+  has_many :credits, dependent: :restrict_with_exception
```

Comment:
An org should never be removed if it has credits

Reply:
I ended up adding this line to my PR for deleting orgs: https://github.com/forem/forem/pull/9357